src/treetensornetworks/opsum_to_ttn.jl

```diff
@@ -821,187 +618,7 @@ function Base.zero(::Type{S}) where {S<:Sum}
-        end_dims = filter(d -> T_inds[d] == -1, dims_out)
-        normal_dims = filter(d -> T_inds[d] != -1, dims_out)
-        T_inds[end_dims] .= linkdims[end_dims] # always end in last channel
-        T_inds[normal_dims] .+= 1 # shift regular channels
-      end
-      T[T_inds...] += t
-    end
-    # add starting and ending identity operators
-    if isnothing(dim_in)
-      T[ones(Int, degrees[v])...] += one(coefficient_type) * Prod([Op("Id", v)]) # only one real starting identity
-    end
-    # ending identities are a little more involved
-    if !isnothing(dim_in)
-      T[linkdims...] += one(coefficient_type) * Prod([Op("Id", v)]) # place identity if all channels end
-      # place identity from start of incoming channel to start of each single outgoing channel
-      idT_end_inds = [linkdims...]
-      idT_end_inds[dim_in] = 1
-      for dout in dims_out
-        idT_end_inds[dout] = 1
-        T[idT_end_inds...] += one(coefficient_type) * Prod([Op("Id", v)])
-        idT_end_inds[dout] = linkdims[dout] # reset
-      end
-    end
-    sparseTTN[v] = T
-  end
-  return sparseTTN, edge_orders
-end
-
-"""
-    fsmTTN(os::OpSum{C}, sites::IndsNetwork{V,<:Index}, root_vertex::V, kwargs...) where {C,V}
-
-Construct a dense TreeTensorNetwork from sparse finite state machine
-represenatation, without compression.
-"""
-function fsmTTN(
-  os::OpSum{C}, sites::IndsNetwork{V,<:Index}, root_vertex::V; trunc=false, kwargs...
-)::TTN where {C,V}
-  coefficient_type = ITensors.determineValType(ITensors.terms(os))
-  # start from finite state machine
-  fsm, edge_orders = finite_state_machine(os, sites, root_vertex)
-  # some trickery to get link dimension for every edge
-  link_space = Dict{edgetype(sites),Index}()
-  function get_linkind!(link_space, e)
-    if !haskey(link_space, e)
-      d = findfirst(x -> (x == e || x == reverse(e)), edge_orders[src(e)])
-      link_space[e] = Index(size(fsm[src(e)], d), edge_tag(e))
-    end
-    return link_space[e]
-  end
-  # compress finite state machine into dense form
-  H = TTN(sites)
-  for v in vertices(sites)
-    linkinds = [get_linkind!(link_space, e) for e in edge_orders[v]]
-    linkdims = dim.(linkinds)
-    H[v] = ITensor()
-    for (T_ind, t) in nonzero_pairs(fsm[v])
-      any(map(x -> abs(coefficient(x)) > eps(), t)) || continue
-      T = zeros(coefficient_type, linkdims...)
-      T[T_ind] += one(coefficient_type)
-      T = itensor(T, linkinds)
-      H[v] += T * computeSiteSum(sites, t)
-    end
-  end
-  # add option for numerical truncation, but throw warning as this can fail sometimes
-  if trunc
-    @warn "Naive numerical truncation of TTN Hamiltonian may fail for larger systems."
-    # see https://github.com/ITensor/ITensors.jl/issues/526
-    lognormT = lognorm(H)
-    H /= exp(lognormT / nv(H)) # TODO: fix broadcasting for in-place assignment
-    H = truncate(H; root_vertex, kwargs...)
-    H *= exp(lognormT / nv(H))
-  end
-  return H
-end
 
 function computeSiteSum(
```

Comment:
Is this being used in the new code? If not it could also be moved to `depracated_opsum_to_ttn.jl`.